module/x/gravity/keeper/ethereum_event_vote.go

```diff
@@ -130,6 +130,7 @@ func (k Keeper) processEthereumEvent(ctx sdk.Context, event types.EthereumEvent)
 			"nonce", fmt.Sprint(event.GetEventNonce()),
 		)
 	} else {
+		ctx.EventManager().EmitEvents(xCtx.EventManager().Events()) // copy events to original context
```

Comment:
Actually, why do we choose to log and ignore the error in the first place? why not just propagate the error?